docs/others/gridnav.rst

```diff
@@ -24,7 +24,7 @@ gridnav is focused, the arrow key presses are automatically forwarded to
 the object so that gridnav can process the arrow keys.
 
 To move the focus to the next widget of the group use
-:c:expr:`LV_KEY_NEXT/PREV` or :c:expr:`lv_group_focus_next/prev()` or the ``TAB``
+:cpp:expr:`LV_KEY_NEXT/PREV` or :cpp:expr:`lv_group_focus_next/prev()` or the ``TAB``
```

Comment:
these an invalid expressions.

This would look better if it read

:cpp:enumerator:\`LV_KEY_NEXT\` or :cpp:enumerator:\`LV_KEY_PREV\`. Optionally you can also use :cpp:func:\`lv_group_focus_next\` or :cpp:func:\`lv_group_focus_prev\`

Reply:
corrected